core/src/graphql/logic/settlePendingSenderTransaction.ts

```diff
@@ -53,15 +52,17 @@ export async function settlePendingSenderTransaction(
       ],
     })
     if (openSenderPendingTx > 1 || openReceiverPendingTx > 1) {
-      throw new LogError('There are more than 1 pending Transactions for Sender and/or Recipient')
+      const errmsg = `There are more than 1 pending Transactions for Sender and/or Recipient`
+      logger.error(errmsg)
+      throw new Error(errmsg)
```

Comment:
Matryoshka error logging

Reply:
Yes thats total correct and should be improved as soon as possible.
At the moment we have no gradido own error-class, which envelopes the original error, to distingue in the upwards throwing-chain if the catched error is still catched before and must not be logged again only throwing or if the catched error is an original error and must be logged and thrown...

So i would preferre to shift this necessary refactoring to a separate issue by change this in an overall effort